setup.py

```diff
@@ -989,6 +996,7 @@ def detect_readline_curses(self):
         elif curses_library == 'curses' and not MACOS:
                 # OSX has an old Berkeley curses, not good enough for
                 # the _curses module.
+            ncurses_enabled = False
```

Comment:
The `ncurses_enabled` vs `curses_enabled` change is hard to follow, and hard to prove won't have unintended side effects on platforms other than AIX. I'd suggest replacing it with a simple `skip_curses_panel` flag that you set to true up on line 910, and then force to false in the AIX branch of the previous if-elif chain.

Then rather than checking for `ncurses_enabled` below, you would instead check for `not skip_curses_panel`.

Reply:
Except, I want to skip_curses_panel if AIX - so maybe it can be even simpler with a statement just before the curses_panel processing.

Comments please.